client/styles/components/_console.scss

```diff
@@ -0,0 +1,27 @@
+.preview-console {
+	position: fixed;
+	width:100%;
+	height:60px;
+	right:0px;
+	bottom: 0px;
+	background:grey;
+	z-index:1000;
+
+	& > {
+		position:relative;
+		text-align:left;
+	}
+
+	// assign styles to different types of console messages
+	.log {
```

Comment:
for this project I'm using BEM naming convention, so these class names should be .preview-console__log, .preview-console__warn, and .preview-console__error. Let's also make these colors the same as the linting error colors, #ffbe05 and #ff5f52. all of the other styling is fine for now.